client/i18n/locales/english/translations.json

```
@@ -377,6 +378,10 @@
     "reset-warn-2": "This cannot be undone",
     "scrimba-tip": "Tip: If the mini-browser is covering the code, click and drag to move it. Also, feel free to stop and edit the code in the video at any time.",
     "chal-preview": "Challenge Preview",
+    "donation-record-not-found": "Your donation record has not been found.",
+    "sign-in-card-update": "Sign in to update your card",
+    "card-has-been-updated": "Your card has been updated successfully.",
+    "contact-support-mistake": "If you think there has been a mistake, please contact support at donors@freecodecamp.org",
```

Given that we have a support email, and don't want general support requests erroneously sent to donors inbox, can we:

```suggestion
    "contact-support-mistake": "If you think there has been a mistake, please contact us at donors@freecodecamp.org",
```

---

Good point